src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -778,9 +778,13 @@
 
   "skyblocker.config.uiAndVisuals.showEquipmentInInventory": "Show Equipment in Inventory",
 
-  "skyblocker.config.uiAndVisuals.tabHud": "Fancy tab HUD (Temporarily disabled outside dungeons)",
+  "skyblocker.config.uiAndVisuals.tabHud": "Fancy HUD and TAB",
+  "skyblocker.config.uiAndVisuals.tabHud.configScreen": "Open Config Screen",
+  "skyblocker.config.uiAndVisuals.tabHud.defaultPositioning": "Default Positioning Behavior",
   "skyblocker.config.uiAndVisuals.tabHud.enableHudBackground": "Enable HUD Background",
   "skyblocker.config.uiAndVisuals.tabHud.enableHudBackground.@Tooltip": "Enables the background of the non-tab HUD.",
+  "skyblocker.config.uiAndVisuals.tabHud.effectsFooter": "Effects from footer",
+  "skyblocker.config.uiAndVisuals.tabHud.effectsFooter.@Tooltip": "If on, will fetch current effects from the tab footer if the Hypixel Effects Widget is disabled.",
   "skyblocker.config.uiAndVisuals.tabHud.nameSorting": "Player Name Sorting Method",
```

Comment:
These translation strings should be deleted since you removed the options (name sorting and plain player names)